analyzers/tests/SonarAnalyzer.UnitTest/Rules/SymbolicExecution/Roslyn/LockReleasedAllPathsTest.cs

```diff
@@ -0,0 +1,88 @@
+﻿/*
```

Comment:
There were rule tests moved to `Sonar` directory. Please move them all back. Rules will be tested in the original classes with different method suffixes.